.goreleaser.yml

```diff
@@ -6,26 +7,53 @@ before:
 builds:
   - env:
       - CGO_ENABLED=0
-    ldflags:
-     - "-X=main.version={{.Env.VERSION}}"
     goos:
       - linux
       - windows
       - darwin
 archives:
-  - replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+  - format: tar.gz
+    # this name template makes the OS and Arch compatible with the results of `uname`.
+    name_template: >-
+      "Bitbucket CLI {{  .Version }}"_
```

Comment:
I think this should be `smug`

Reply:
Same as above. It turns out my branch was out of sync from master. I have pushed a new commit